core/src/main/resources/hudson/logging/Messages.properties

```diff
@@ -20,5 +20,5 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-LogRecorderManager.init=Initialing log recorders
+LogRecorderManager.init=Initializing log recorders
```

Comment:
Good catch :)

Reply:
Usually an unrelated change like this would be better off in a separate PR, but I guess it's small enough that the overhead isn't worth it.